hack/install-kubestellar.sh

```diff
@@ -19,11 +19,12 @@
 # This script installs Kubestellar binaries to a folder of choice
 #
 # Arguments:
-# [--version release_version] set a specific Kubestellar release version, default: latest
+# [--version release] set a specific Kubestellar release version, default: latest
 # [--os linux|darwin] set a specific OS type, default: autodetect
 # [--arch amd64|arm64] set a specific architecture type, default: autodetect
-# [--folder installation_folder] sets the installation folder, default: $PWD/kubestellar
+# [--folder name] sets the installation folder, default: $PWD/kubestellar
 # [--create-folder] create the instllation folder, if it does not exist
+# [--strip-bin] remove the bin sub-folder
```

Comment:
I do not think we need or want this `strip-bin` option.  Since our archives will contain `example/` and `config/` as well as `bin/`, it would be odd to strip just the `bin` (and confusing to merge all three together).